exercises/hello-world/examples/success-standard/src/HelloWorld.hs

```diff
@@ -0,0 +1,4 @@
+module HelloWorld (hello) where
+
+hello :: String
+hello = "Hello, World!"
```

Comment:
I'm not sure I understand. Isn't this (putting the sample in an `examples` directory) the way other exercises also work?